source/code-snippets/usage-examples/count.js

```diff
@@ -15,16 +21,17 @@ async function run() {
     const estimate = await movies.estimatedDocumentCount();
     console.log(`Estimated number of documents in the movies collection: ${estimate}`);
 
-    // Query for movies from Canada.
+    // Query for movies where the countries field includes "Canada".
     const query = { countries: "Canada" };
 
     // Find the number of documents that match the specified
-    // query, (i.e. with "Canada" as a value in the "countries" field)
-    // and print out the count.
+    // query and print out the count.
```

Comment:
Suggestion:
I think for multi-line comments, it would be good to use block-style comments "/* */"

I think this could be simplified to:
"Print the number of documents that match the query."